src/applications/edu-benefits/0994/pages/bankInformation.js

```diff
@@ -31,13 +44,17 @@ export const uiSchema = {
 export const schema = {
   type: 'object',
   properties: {
-    bankAccount: {
+    'view:bankAccount': {
       type: 'object',
-      required: ['accountType', 'routingNumber', 'accountNumber'],
       properties: {
-        accountType,
-        routingNumber,
-        accountNumber,
+        bankAccount: {
```

Comment:
this can just be imported from `vets-json-schema/dist/22-0994-schema.json`.

i.e. just do `bankAccount`